documentation/collectionruleexamples.md

```diff
@@ -33,16 +33,37 @@ The following examples provide sample scenarios for using a collection rule. The
 </details>
 
 <details>
-  <summary>Environment Variables</summary>
+  <summary>ConfigMap</summary>
   
   ```bash
```

Comment:
Can you see if there's a ```yaml?

Reply:
There is, didn't even think of it - I'll switch to that for ConfigMap and the Kubernetes Environment Variables